[
    {
        "id": "24791385791",
        "type": "IssuesEvent",
        "actor": {
            "id": 57632,
            "login": "todo",
            "display_login": "todo",
            "gravatar_id": "",
            "url": "https://api.github.com/users/todo",
            "avatar_url": "https://avatars.githubusercontent.com/u/57632?"
        },
        "repo": {
            "id": 26376490,
            "name": "veracrypt/VeraCrypt",
            "url": "https://api.github.com/repos/veracrypt/VeraCrypt"
        },
        "payload": {
            "action": "opened",
            "issue": {
                "url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991",
                "repository_url": "https://api.github.com/repos/veracrypt/VeraCrypt",
                "labels_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/labels{/name}",
                "comments_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/comments",
                "events_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/events",
                "html_url": "https://github.com/veracrypt/VeraCrypt/issues/991",
                "id": 1420884586,
                "node_id": "I_kwDOAZJ5Ks5UsPpq",
                "number": 991,
                "title": "mounted exFAT file container not responsive ",
                "user": {
                    "login": "todo",
                    "id": 57632,
                    "node_id": "MDQ6VXNlcjU3NjMy",
                    "avatar_url": "https://avatars.githubusercontent.com/u/57632?v=4",
                    "gravatar_id": "",
                    "url": "https://api.github.com/users/todo",
                    "html_url": "https://github.com/todo",
                    "followers_url": "https://api.github.com/users/todo/followers",
                    "following_url": "https://api.github.com/users/todo/following{/other_user}",
                    "gists_url": "https://api.github.com/users/todo/gists{/gist_id}",
                    "starred_url": "https://api.github.com/users/todo/starred{/owner}{/repo}",
                    "subscriptions_url": "https://api.github.com/users/todo/subscriptions",
                    "organizations_url": "https://api.github.com/users/todo/orgs",
                    "repos_url": "https://api.github.com/users/todo/repos",
                    "events_url": "https://api.github.com/users/todo/events{/privacy}",
                    "received_events_url": "https://api.github.com/users/todo/received_events",
                    "type": "User",
                    "site_admin": false
                },
                "labels": [],
                "state": "open",
                "locked": false,
                "assignee": null,
                "assignees": [],
                "milestone": null,
                "comments": 0,
                "created_at": "2022-10-24T14:00:44Z",
                "updated_at": "2022-10-24T14:00:44Z",
                "closed_at": null,
                "author_association": "NONE",
                "active_lock_reason": null,
                "body": "background info:\r\n- 8GB exFAT file container is stored on Synology NAS\r\n- mounting the file container will work for awhile, then become non-responsive after some period of no operations to/from the volume\r\n- Windows 11 Home, on LAN network\r\n- Veracrypt 1.25.9 (64bit)\r\n\r\nI cannot figured out how long of 'no operations' will cause the volume to be non-responsive. After it becomes non-responsive, I'll get the error \"0x8007003B: an unexpected network error occurred\" when accessing any files. However, I can still scroll through the directories, but I just cannot open any files for read/write.\r\n\r\n1. Is this problem caused by Synology's Samba driver?\r\n2. Or is it an exFAT filesystem problem?\r\n3. Or will switching to NTFS fix this?\r\n4. Or is it a Veracrypt's mounting issue?\r\n\r\nThanks!",
                "reactions": {
                    "url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/reactions",
                    "total_count": 0,
                    "+1": 0,
                    "-1": 0,
                    "laugh": 0,
                    "hooray": 0,
                    "confused": 0,
                    "heart": 0,
                    "rocket": 0,
                    "eyes": 0
                },
                "timeline_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/timeline",
                "performed_via_github_app": null,
                "state_reason": null
            }
        },
        "public": true,
        "created_at": "2022-10-24T14:00:44Z",
        "org": {
            "id": 7925881,
            "login": "veracrypt",
            "gravatar_id": "",
            "url": "https://api.github.com/orgs/veracrypt",
            "avatar_url": "https://avatars.githubusercontent.com/u/7925881?"
        }
    },
    {
        "id": "24791385791",
        "type": "IssuesEvent",
        "actor": {
            "id": 57632,
            "login": "todo",
            "display_login": "todo",
            "gravatar_id": "",
            "url": "https://api.github.com/users/todo",
            "avatar_url": "https://avatars.githubusercontent.com/u/57632?"
        },
        "repo": {
            "id": 26376490,
            "name": "veracrypt/VeraCrypt",
            "url": "https://api.github.com/repos/veracrypt/VeraCrypt"
        },
        "payload": {
            "action": "opened",
            "issue": {
                "url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991",
                "repository_url": "https://api.github.com/repos/veracrypt/VeraCrypt",
                "labels_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/labels{/name}",
                "comments_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/comments",
                "events_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/events",
                "html_url": "https://github.com/veracrypt/VeraCrypt/issues/991",
                "id": 1420884586,
                "node_id": "I_kwDOAZJ5Ks5UsPpq",
                "number": 991,
                "title": "mounted exFAT file container not responsive ",
                "user": {
                    "login": "todo",
                    "id": 57632,
                    "node_id": "MDQ6VXNlcjU3NjMy",
                    "avatar_url": "https://avatars.githubusercontent.com/u/57632?v=4",
                    "gravatar_id": "",
                    "url": "https://api.github.com/users/todo",
                    "html_url": "https://github.com/todo",
                    "followers_url": "https://api.github.com/users/todo/followers",
                    "following_url": "https://api.github.com/users/todo/following{/other_user}",
                    "gists_url": "https://api.github.com/users/todo/gists{/gist_id}",
                    "starred_url": "https://api.github.com/users/todo/starred{/owner}{/repo}",
                    "subscriptions_url": "https://api.github.com/users/todo/subscriptions",
                    "organizations_url": "https://api.github.com/users/todo/orgs",
                    "repos_url": "https://api.github.com/users/todo/repos",
                    "events_url": "https://api.github.com/users/todo/events{/privacy}",
                    "received_events_url": "https://api.github.com/users/todo/received_events",
                    "type": "User",
                    "site_admin": false
                },
                "labels": [],
                "state": "open",
                "locked": false,
                "assignee": null,
                "assignees": [],
                "milestone": null,
                "comments": 0,
                "created_at": "2022-10-24T14:00:44Z",
                "updated_at": "2022-10-24T14:00:44Z",
                "closed_at": null,
                "author_association": "NONE",
                "active_lock_reason": null,
                "body": "background info:\r\n- 8GB exFAT file container is stored on Synology NAS\r\n- mounting the file container will work for awhile, then become non-responsive after some period of no operations to/from the volume\r\n- Windows 11 Home, on LAN network\r\n- Veracrypt 1.25.9 (64bit)\r\n\r\nI cannot figured out how long of 'no operations' will cause the volume to be non-responsive. After it becomes non-responsive, I'll get the error \"0x8007003B: an unexpected network error occurred\" when accessing any files. However, I can still scroll through the directories, but I just cannot open any files for read/write.\r\n\r\n1. Is this problem caused by Synology's Samba driver?\r\n2. Or is it an exFAT filesystem problem?\r\n3. Or will switching to NTFS fix this?\r\n4. Or is it a Veracrypt's mounting issue?\r\n\r\nThanks!",
                "reactions": {
                    "url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/reactions",
                    "total_count": 0,
                    "+1": 0,
                    "-1": 0,
                    "laugh": 0,
                    "hooray": 0,
                    "confused": 0,
                    "heart": 0,
                    "rocket": 0,
                    "eyes": 0
                },
                "timeline_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/timeline",
                "performed_via_github_app": null,
                "state_reason": null
            }
        },
        "public": true,
        "created_at": "2022-10-24T14:00:44Z",
        "org": {
            "id": 7925881,
            "login": "veracrypt",
            "gravatar_id": "",
            "url": "https://api.github.com/orgs/veracrypt",
            "avatar_url": "https://avatars.githubusercontent.com/u/7925881?"
        }
    },
    {
        "id": "24791385791",
        "type": "IssuesEvent",
        "actor": {
            "id": 57632,
            "login": "todo",
            "display_login": "todo",
            "gravatar_id": "",
            "url": "https://api.github.com/users/todo",
            "avatar_url": "https://avatars.githubusercontent.com/u/57632?"
        },
        "repo": {
            "id": 26376490,
            "name": "veracrypt/VeraCrypt",
            "url": "https://api.github.com/repos/veracrypt/VeraCrypt"
        },
        "payload": {
            "action": "opened",
            "issue": {
                "url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991",
                "repository_url": "https://api.github.com/repos/veracrypt/VeraCrypt",
                "labels_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/labels{/name}",
                "comments_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/comments",
                "events_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/events",
                "html_url": "https://github.com/veracrypt/VeraCrypt/issues/991",
                "id": 1420884586,
                "node_id": "I_kwDOAZJ5Ks5UsPpq",
                "number": 991,
                "title": "mounted exFAT file container not responsive ",
                "user": {
                    "login": "todo",
                    "id": 57632,
                    "node_id": "MDQ6VXNlcjU3NjMy",
                    "avatar_url": "https://avatars.githubusercontent.com/u/57632?v=4",
                    "gravatar_id": "",
                    "url": "https://api.github.com/users/todo",
                    "html_url": "https://github.com/todo",
                    "followers_url": "https://api.github.com/users/todo/followers",
                    "following_url": "https://api.github.com/users/todo/following{/other_user}",
                    "gists_url": "https://api.github.com/users/todo/gists{/gist_id}",
                    "starred_url": "https://api.github.com/users/todo/starred{/owner}{/repo}",
                    "subscriptions_url": "https://api.github.com/users/todo/subscriptions",
                    "organizations_url": "https://api.github.com/users/todo/orgs",
                    "repos_url": "https://api.github.com/users/todo/repos",
                    "events_url": "https://api.github.com/users/todo/events{/privacy}",
                    "received_events_url": "https://api.github.com/users/todo/received_events",
                    "type": "User",
                    "site_admin": false
                },
                "labels": [],
                "state": "open",
                "locked": false,
                "assignee": null,
                "assignees": [],
                "milestone": null,
                "comments": 0,
                "created_at": "2022-10-24T14:00:44Z",
                "updated_at": "2022-10-24T14:00:44Z",
                "closed_at": null,
                "author_association": "NONE",
                "active_lock_reason": null,
                "body": "background info:\r\n- 8GB exFAT file container is stored on Synology NAS\r\n- mounting the file container will work for awhile, then become non-responsive after some period of no operations to/from the volume\r\n- Windows 11 Home, on LAN network\r\n- Veracrypt 1.25.9 (64bit)\r\n\r\nI cannot figured out how long of 'no operations' will cause the volume to be non-responsive. After it becomes non-responsive, I'll get the error \"0x8007003B: an unexpected network error occurred\" when accessing any files. However, I can still scroll through the directories, but I just cannot open any files for read/write.\r\n\r\n1. Is this problem caused by Synology's Samba driver?\r\n2. Or is it an exFAT filesystem problem?\r\n3. Or will switching to NTFS fix this?\r\n4. Or is it a Veracrypt's mounting issue?\r\n\r\nThanks!",
                "reactions": {
                    "url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/reactions",
                    "total_count": 0,
                    "+1": 0,
                    "-1": 0,
                    "laugh": 0,
                    "hooray": 0,
                    "confused": 0,
                    "heart": 0,
                    "rocket": 0,
                    "eyes": 0
                },
                "timeline_url": "https://api.github.com/repos/veracrypt/VeraCrypt/issues/991/timeline",
                "performed_via_github_app": null,
                "state_reason": null
            }
        },
        "public": true,
        "created_at": "2022-10-24T14:00:44Z",
        "org": {
            "id": 7925881,
            "login": "veracrypt",
            "gravatar_id": "",
            "url": "https://api.github.com/orgs/veracrypt",
            "avatar_url": "https://avatars.githubusercontent.com/u/7925881?"
        }
    }
]